gradle-jdks/src/test/groovy/com/palantir/gradle/jdks/JdksPluginIntegrationSpec.groovy

```diff
@@ -61,6 +66,76 @@ class JdksPluginIntegrationSpec extends IntegrationSpec {
+            MIIDQTCCAimgAwIBAgITBmyfz5m/jAo54vB4ikPmljZbyjANBgkqhkiG9w0BAQsF
+            ADA5MQswCQYDVQQGEwJVUzEPMA0GA1UEChMGQW1hem9uMRkwFwYDVQQDExBBbWF6
+            b24gUm9vdCBDQSAxMB4XDTE1MDUyNjAwMDAwMFoXDTM4MDExNzAwMDAwMFowOTEL
+            MAkGA1UEBhMCVVMxDzANBgNVBAoTBkFtYXpvbjEZMBcGA1UEAxMQQW1hem9uIFJv
+            b3QgQ0EgMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALJ4gHHKeNXj
+            ca9HgFB0fW7Y14h29Jlo91ghYPl0hAEvrAIthtOgQ3pOsqTQNroBvo3bSMgHFzZM
+            9O6II8c+6zf1tRn4SWiw3te5djgdYZ6k/oI2peVKVuRF4fn9tBb6dNqcmzU5L/qw
+            IFAGbHrQgLKm+a/sRxmPUDgH3KKHOVj4utWp+UhnMJbulHheb4mjUcAwhmahRWa6
+            VOujw5H5SNz/0egwLX0tdHA114gk957EWW67c4cX8jJGKLhD+rcdqsq08p8kDi1L
+            93FcXmn/6pUCyziKrlA4b9v7LWIbxcceVOF34GfID5yHI9Y/QCB/IIDEgEw+OyQm
+            jgSubJrIqg0CAwEAAaNCMEAwDwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMC
+            AYYwHQYDVR0OBBYEFIQYzIU07LwMlJQuCFmcx7IQTgoIMA0GCSqGSIb3DQEBCwUA
+            A4IBAQCY8jdaQZChGsV2USggNiMOruYou6r4lK5IpDB/G/wkjUu0yKGX9rbxenDI
+            U5PMCCjjmCXPI6T53iHTfIUJrU6adTrCC2qJeHZERxhlbI1Bjjt/msv0tadQ1wUs
+            N+gDS63pYaACbvXy8MWy7Vu33PqUXHeeE6V/Uq2V8viTO96LXFvKWlJbYK8U90vv
+            o/ufQJVtMVT8QtPHRh8jrdkPSHCa2XV4cdFyQzR1bldZwgJcJmApzyMZFo6IQ6XU
+            5MsI+yMRQ+hDKXJioaldXgjUkK642M4UwtBV8ob2xJNDd2ZhwLnoQdeXeGADbkpy
+            rqXRfboQnoZsG4q5WTP468SQvvG5
+            -----END CERTIFICATE-----
+        '''.stripIndent(true)
+
+
+        // language=gradle
+        buildFile << '''
+            jdks {
+                caCerts.put 'Our_Amazon_CA_Cert_1', file('amazon-root-ca-1.pem')
+            }
+
+            task printCaTruststoreAliases(type: JavaExec) {
+                classpath = sourceSets.main.runtimeClasspath
+                mainClass = 'foo.OutputCaCerts'
+                logging.captureStandardOutput LogLevel.LIFECYCLE
+                logging.captureStandardError LogLevel.LIFECYCLE
+            }
+        '''.stripIndent(true)
+
+        // language=java
+        writeJavaSourceFile '''
+            package foo;
+
+            import java.io.File;
+            import java.security.KeyStore;import java.security.cert.X509Certificate;
+            
+            public final class OutputCaCerts {
+                public static void main(String... args) throws Exception {
+                    KeyStore keyStore = KeyStore.getInstance(
+                            new File(System.getProperty("java.home"), "lib/security/cacerts"),
+                            "changeit".toCharArray());
+                    System.out.println(
+                            ((X509Certificate) keyStore.getCertificate("our_amazon_ca_cert_1")).getSerialNumber());
+                }
+            }
+        '''.stripIndent(true)
+
+        when:
+
+        def stdout = runTasksSuccessfully(
+                'printCaTruststoreAliases',
+                // TODO: avoid resolving from root configuration somehow, removed in Gradle 8
+                '--warning-mode=none')
```

Comment:
Sounds like we may not need to suppress warnings here either?
```suggestion
                'printCaTruststoreAliases')
```

Reply:
👍🏻 yep, will fix it up with the conflicts